pkg/install/install.go

```diff
@@ -192,7 +192,6 @@ func (i *Installer) Install(ctx context.Context, installConfig *installconfig.In
 			action(i.configureAPIServerCertificate),
 			condition{i.apiServersReady, 30 * time.Minute},
 			condition{i.operatorConsoleExists, 30 * time.Minute},
-			action(i.updateConsoleBranding),
```

Comment:
as 4.4 removed customization, do we want to keep this action? Do we know if this is coming back?

Reply:
Ok. It was the change in console itslef (ref https://github.com/openshift/console/pull/4954, backported to 4.4 in https://github.com/openshift/console/pull/4992).

@mjudeikis despite the fact that the logo is generic OpenShift logo, I think we still need to apply the branding change: it changes the product name (affects page titles and probably some other bits).

Reply:
Done.